Resources/Catalog/FileCatalogClient.py

```diff
@@ -292,7 +292,7 @@ def getFileUserMetadata(self, path, timeout=120):
     """
     directory = "/".join(path.split("/")[:-1])
     rpcClient = self._getRPC(timeout=timeout)
-    result = rpcClient.getFileUserMetadata(path)
+    result = rpcClient.findDirectoriesByMetadata(path)
```

Comment:
Is it intentional ?

Reply:
Not sure why this happened. It clearly look like my commit.